docs/tutorials/contract-deployment.md

```diff
@@ -0,0 +1,93 @@
+- [Approved Kovan price identifiers](https://thegraph.com/explorer/subgraph/umaprotocol/uma-kovan?query=Pricefeed%20Identifiers)
+
+To add a new price identifier or collateral currency to mainnet, please propose one to UMA voters by following the [UMIP process](https://docs.umaproject.org/governance/umips).
+
+To add a new price identifier or collateral currency to Kovan, please follow these [instructions](https://docs.umaproject.org/governance/adding-price-id#adding-a-price-identifier-to-the-kovan-testnet) or contact the UMA core team in Discord.
+
+## EMP parameters
+
+The mandatory EMP parameters are:
+
+- `expirationTimestamp`: The UNIX timestamp that the EMP will expire at. Example December 31, 2024 expiry: `1735682400`.
+- `collateralAddress`: The token address of the Kovan or mainnet token that will be used as the collateral currency. Example Kovan WETH address: `0xd0A1E359811322d97991E03f863a0C30C2cF029C`.
+- `priceFeedIdentifier`: The plaintext price identifier converted to bytes32 format. Example for USDETH: `0x555344455448`.
+- `syntheticName`: The plaintext synthetic token name. Example: `Synthetic Test USDETH Dec 2024`.
+- `syntheticSymbol`: The plaintext synthetic token symbol. Example: `USDETH-DEC-2024`.
+- `collateralRequirement`: The [collateralization requirement ratio](https://docs.umaproject.org/synthetic-tokens/glossary#collateralization-requirement) converted to Wei. Example for a 1.5 ratio: `{ rawValue: '1500000000000000000' }`.
+- `disputeBondPct`: The percentage size of the [dispute bond](https://docs.umaproject.org/synthetic-tokens/explainer#liquidation-and-dispute) converted to 18 decimals. Example for 10%: `{ rawValue: '100000000000000000' }`.
+- `sponsorDisputeRewardPct`: The percentage size of the [dispute reward](https://docs.umaproject.org/synthetic-tokens/explainer#liquidation-and-dispute) paid to the position sponsor converted to 18 decimals. Example for 5%: `{ rawValue: '50000000000000000' }`.
+- `disputerDisputeRewardPct`: The percentage size of the dispute reward paid to the disputer converted to 18 decimals. Example for 20%: `{ rawValue: '200000000000000000' }`.
+- `minSponsorTokens`: The minimum number of tokens required in a sponsor position converted to 18 decimals. Example for 100 tokens: `{ rawValue: '100000000000000000000' }`.
+- `withdrawalLiveness`: The length of the [withdrawal liveness period](https://docs.umaproject.org/synthetic-tokens/glossary#withdrawal-liveness-period) in seconds. Example for 2 hours: `7200`.
+- `liquidationLiveness`: The length of the [liquidation liveness period](https://docs.umaproject.org/synthetic-tokens/glossary#liquidation-liveness-period) in seconds. Example for 2 hours: `7200`.
+- `excessTokenBeneficiary`: The Ethereum address that will receive any sort of excess collateral accrued by interest bearing collateral currencies, such as rDAI.
+
+This is what the parameters object will look like when correctly formatted.
+
+```json
+{   
```

Comment:
does this need to be wrapped into a ` punctuation mark or something for etherscan support?

Reply:
Nope - it's just the object with no wrapping punctuation